parso/grammar.py

```diff
@@ -216,45 +224,44 @@ def __init__(self, version_info, bnf_text):
         self.version_info = version_info
 
     def _tokenize_lines(self, lines, **kwargs):
-        return tokenize_lines(lines, self.version_info, **kwargs)
+        return tokenize_lines(lines, version_info=self.version_info, **kwargs)
 
     def _tokenize(self, code):
         # Used by Jedi.
-        return tokenize(code, self.version_info)
+        return tokenize(code, version_info=self.version_info)
 
 
-def load_grammar(**kwargs):
+def load_grammar(*, language: str = 'python', version: str = None, path: str = None):
```

Comment:
I don't see the point of this `language` feature, at least for now. Would you mind briefly explaining it to me?

Reply:
Absolutely. Removed.